deploy/example_clusters/cr-cluster-cdh632cm-stor.yaml

```diff
@@ -0,0 +1,104 @@
+apiVersion: "kubedirector.hpe.com/v1beta1"
+kind: "KubeDirectorCluster"
+metadata:
+  name: "cdh632-cm-persistent"
+spec:
+  app: cdh632
+  roles:
+  - id: cmserver
+    resources:
+      requests:
+        memory: "8Gi"
+        cpu: "4"
+      limits:
```

Comment:
Are the memory and CPU requirements the same as we did for EPIC ?

Reply:
@kmathur2 
Nope. The cardinality is 1 for apps in EPIC. The thing is, in EPIC, the default selected roles are ["cmserver", "primary", "hive", "worker", "gateway"]. We bring in other roles when HA is selected. So we, kind of, tie together HA and installation of other roles. I don't know whether that is the right thing to do.

Reply:
@kmathur2 Yes, the memory and cpu  requirements for the roles is in accordance with min_cores and min_memory of the json file in EPIC.